Routing/Loader/DirectoryRouteLoader.php

```diff
@@ -50,8 +50,10 @@ public function load($resource, $type = null)
         $finder = new Finder();
 
         foreach ($finder->in($resource)->name('*.php')->files() as $file) {
-            $imported = $this->processor->importResource($this, ClassUtils::findClassInFile($file), array(), null, null, 'rest');
-            $collection->addCollection($imported);
+            if($class = ClassUtils::findClassInFile($file)) {
```

Comment:
I'm really wondering in what cases that might happen, but sure, it would be, technically, more correct :)

What do you prefer, assign in a separate line then check or "&& $class !== false"?